HermaFx.Rebus/SagaPersisters/DualSagaPersister.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Rebus;
+
+namespace HermaFx.Rebus
+{
+	public class DualSagaPersister : IStoreSagaData
+	{
+		private const string DUAL_SAGA_ID = "HermaFx.Rebus:Dual_Saga_";
+
```

Comment:
This ain't an ID.. What about:

private const string SAGA_PERSISTER_IMPL = nameof(DualSagaPersister) + ":PersisterImpl:";

?